spec/person_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new(20, 'Tati')
+  end
+
+  describe '#new' do
+    it 'takes three parameters and returns a Person object' do
+      expect(@person).to be_an_instance_of Person
+    end
+  end
+
+  describe '#name' do
+    it 'returns the correct name' do
+      expect(@person.name).to eql('Tati')
+    end
+  end
+
+  describe '#age' do
+    it 'returns the correct age' do
+      expect(@person.age).to eql(20)
+    end
+  end
+
+  describe '#can_use_services?' do
+    it 'should return true' do
+      expect(@person.can_use_services?).to be true
+    end
+  end
```

Comment:
- [ ] Good job adding these tests, however, to ensure you are covering all case scenarios, kindly also check for some positive and negative case scenarios e.g:
```The can_use_services? method returns false if under age and no parent permission```
```The can_use_services? method returns true if over age and no parent permission```